fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -209,6 +209,10 @@ class FullyShardedDataParallel(nn.Module):
         verbose (bool):
             Set this to ``True`` to turn on verbose output for model's string representation.
             Default: False
+        cpu_offload (bool, Optional):
```

Comment:
how about
```suggestion
        # deprecated args below.
        cpu_offload (bool, Optional):
```

Reply:
I think I have note in the description so I don't think we need this comment. Feel free to change that in your upcoming refactor if you feel strongly.